### ISSUE TITLE
struct.error

### ISSUE THREAD

**Issue by king-menin** [ptillet/torch-blocksparse#27]
Do you have any suggestions why the error might be in softmax?
`struct.error: 'I' format requires 0 <= number <= 4294967295`
I train my model, stop training, load model and resume training and have this error :-(
Actually i reimplement mha at my model with your functions and torch_blocksparse.MatMul worked well

**Comment by ptillet** [comment]
It seems like there's an int argument of the Triton kernel that overflows. What are the matrix dimensions?

**Comment by king-menin** [comment]
query size = [bs=1, seq_len=2048, emb_dim=1536]. sparsity.stride=1024, sparsity.block = 16
I suppose It was memory issue. TY!
